### ISSUE TITLE
[Bug][L10N] Improperly formatted CS (czech) string 

### ISSUE THREAD

**Issue by liuche** [mozilla-mobile/fenix#6959]
There's a translation that's failing the Android [lint check](https://firefox-ci-tc.services.mozilla.com/tasks/cyf4ln86Qa6_HKNfiRf5Fw/runs/0/logs/https%3A%2F%2Ffirefox-ci-tc.services.mozilla.com%2Fapi%2Fqueue%2Fv1%2Ftask%2Fcyf4ln86Qa6_HKNfiRf5Fw%2Fruns%2F0%2Fartifacts%2Fpublic%2Flogs%2Flive.log).

% must be escaped by using %% instead, in the "search?q=%s" portion of the text.

```
[task 2019-12-05T04:13:15.747Z]   /builds/worker/checkouts/src/app/src/main/res/values-cs/strings.xml:1061: Error: Incorrect formatting string search_add_custom_engine_search_string_example; missing conversion character in '% s' ? [StringFormatInvalid]
[task 2019-12-05T04:13:15.747Z]       <string name="search_add_custom_engine_search_string_example">Dotaz nahraďte „% s“, Příklad: \nhttps://www.google.com/search?q=%s</string>
```

@Delphine where should we open bugs for translations?

**Comment by mcarare** [comment]
@liuche This was corrected in Pontoon: https://pontoon.mozilla.org/cs/android-l10n/mozilla-mobile/fenix/app/src/main/res/values/strings.xml/?search=add_custom&string=205607
I also had a fix in https://github.com/mozilla-mobile/fenix/pull/6963, but I think it will be ok in the next l10n import without merging my PR.

**Comment by liuche** [comment]
Closed by #6963, and also fixed in pontoon.
